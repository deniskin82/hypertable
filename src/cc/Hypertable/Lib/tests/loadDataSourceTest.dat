#timestamp	rowkey	columnkey	value
2009-02-12 02:12:12.1000000000	abenteric	foo:http://www.foo.com/	How now brown cow
2009-02-12 02:12:12:2	barothermohygrograph	bar	Herbie Hancock
2009-02-12 02:12:12	Batonga	foo:	Head Hunters
2009-02-12 02:12:12
abracadabra	Batonga	foo:	Head Hunters
2009-02-12 02:12:12	barothermohygrogra
2009-02-12 02:12:12	barothermohygrograph	ba
2009-02-12 02:12:12.1000000000	carbonification	sherlock:holmes	this\tis\ta\ttab\tdelimited\tvalue
2009-02-12 02:12:12.3000000000	biomagnify	sherlock:holmes	\\\this\contains\lots\of\backslashes\\\
2009-02-12 02:12:13	dandy	foo	chicken
2009-02-12 02:12:13	dandy	bar	chicken
2009-02-12 02:12:13	warhols	foo:a	scratch
2009-02-12 02:12:13	warhols	foo:b	scratch
2009-02-12 02:12:13	warhols	bar	scratch
2009-02-12 02:12:13	warhols	bar:a	scratch
2009-02-12 02:12:13	warhols	bar:b	scratch
2009-02-12 02:12:13	dandy			DELETE
2009-02-12 02:12:13	warhols	bar		DELETE
2009-02-12 02:12:13	warhols	foo:a		DELETE


