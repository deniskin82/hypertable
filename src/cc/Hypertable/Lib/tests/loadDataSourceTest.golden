row=abenteric	column_family=foo	column_qualifier=http://www.foo.com/	value=How now brown cow
row=barothermohygrograph	column_family=bar	value=Herbie Hancock
row=Batonga	column_family=foo	value=Head Hunters
warning: too few fields on line 5
warn: invalid timestamp format on line 6, skipping...
warning: too few fields on line 7
warning: too few fields on line 8
row=carbonification	column_family=sherlock	column_qualifier=holmes	value=this\tis\ta\ttab\tdelimited\tvalue
row=biomagnify	column_family=sherlock	column_qualifier=holmes	value=\\\this\contains\lots\of\backslashes\\\
row=dandy	column_family=foo	value=chicken
row=dandy	column_family=bar	value=chicken
row=warhols	column_family=foo	column_qualifier=a	value=scratch
row=warhols	column_family=foo	column_qualifier=b	value=scratch
row=warhols	column_family=bar	value=scratch
row=warhols	column_family=bar	column_qualifier=a	value=scratch
row=warhols	column_family=bar	column_qualifier=b	value=scratch
row=dandy	value=	DELETE
row=warhols	column_family=bar	value=	DELETE
row=warhols	column_family=foo	column_qualifier=a	value=	DELETE
warning: too few fields on line 21
warning: too few fields on line 22
